docs/tips-for-windows-dev.md

```diff
@@ -47,7 +43,7 @@ copy C:\ProgramData\Multipass\data\ssh-keys\id_rsa %HOMEDRIVE%%HOMEPATH%\.ssh\mu
 ```
 Host fabric-dev
   HostName 172.31.118.103
-  User ubuntu
+  User dev
```

Comment:
BTW I did change the multipass user to "ubuntu" in a PR yesterday per Matthew's suggestion. So should this remain "ubuntu"?

Reply:
you're right @denyeart...